packages/data-stores/src/plans/types.ts

```diff
@@ -57,7 +57,7 @@ export interface PricedAPIPlan {
 	path_slug?: PlanPath;
 	product_slug: StorePlanSlug;
 	product_name_short: string;
-	bill_period: -1 | 31 | 365;
+	bill_period: -1 | 31 | 365 | 730;
```

Comment:
> I'm also wondering if we can somehow unify this with the [PERIOD_LIST](https://github.com/Automattic/wp-calypso/blob/trunk/packages/calypso-products/src/constants/terms.ts#L13) here 🤔

I am not sure if we can do that right now. `data-stores` is on [NPM](https://www.npmjs.com/package/@automattic/data-stores), while `calypso-products` hasn't been published yet. We can use `workspace` dependencies, which (unless I am mistaken) will pick the dependency off the Calypso repo, but that will probably create issues if a new version is pushed to NPM. I think..

but also wondering if thinking in terms of coupling `data-stores/plans` with `calypso-prodicts/plans` would be ideal long term. I'd think the former should come to replace the latter entirely at some point i.e. `data-stores/plans` being enough, so pros and cons I guess

> If not, could we replace 31, 365, 730 by the same set of constants?

Yes, this should be fine for now. There are places in the package where these values are being used. Happy to introduce the constants and use them for this instance, but maybe we can handle a wider refactor in some other PR. We may need to deep dive a little more to test this PR as a bigger `data-stores` refactor 🤔

Reply:
> I am not sure if we can do that right now. data-stores is on [NPM](https://www.npmjs.com/package/@automattic/data-stores), while calypso-products hasn't been published yet. We can use workspace dependencies, which (unless I am mistaken) will pick the dependency off the Calypso repo, but that will probably create issues if a new version is pushed to NPM. I think..

Agreed. Introducing dependency between `data-stores/plans` and `calypso-products/plans` sounds muddy, since they feel roughly at the same level of the whole dependency hierarchy. Let's not do that to limit the scope of this PR.

> maybe we can handle a wider refactor in some other PR. We may need to deep dive a little more to test this PR as a bigger data-stores refactor 🤔

Also agreed. Let's keep the scope of this PR tangible :)